src/facet.scss

```diff
@@ -0,0 +1,167 @@
+@import "./new-settings";
+@import "./mixins";
+
+$block: #{$fd-namespace}-facet;
+
+.#{$block} {
+  $fd-facet-horizontal-spacing: 2rem;
+  $fd-facet-vertical-spacing: 1rem;
+  $fd-facet-title-vertical-spacing: 1rem;
+  $fd-form-kv-facet-horizontal-spacing: 0.5rem;
+  $fd-image-form-facet-bottom-spacing: 0.5rem;
+  $fd-image-facet-mobile-spacing: 1rem;
+  $fd-image-facet-header-horizontal-spacing: 1rem;
+  $fd-object-status-facet-font-size: 1.5rem;
+
+  @mixin facet-spacing() {
+    margin-right: $fd-facet-horizontal-spacing;
+    margin-bottom: $fd-facet-vertical-spacing;
+
+    @include fd-rtl() {
+      margin-right: 0;
+      margin-left: $fd-facet-horizontal-spacing;
+    }
+  }
+
+  @mixin image-facet-mobile-spacing() {
+    margin-right: $fd-image-facet-mobile-spacing;
+    margin-bottom: $fd-image-form-facet-bottom-spacing;
+
+    @include fd-rtl() {
+      margin-right: 0;
+      margin-left: $fd-image-facet-mobile-spacing;
+    }
+  }
+
+  @include fd-reset();
+
+  @include fd-flex() {
+    flex-wrap: wrap;
+  }
+
+  &__container {
+    @include fd-reset();
+    @include facet-spacing();
+
+    &--image {
+      @include fd-flex();
+
+      @media (min-width: 0) {
+        @include image-facet-mobile-spacing();
+      }
+
+      @media (min-width: 37rem) {
+        @include facet-spacing();
+      }
+
+      // when used in object/dynamic page header
+      &-header {
+        margin-right: $fd-image-facet-header-horizontal-spacing;
+        margin-bottom: 0;
+        align-items: center;
+
+        @include fd-rtl() {
+          margin-right: 0;
+          margin-left: $fd-image-facet-header-horizontal-spacing;
+        }
+      }
+
+      // when used in object/dynamic page header in mobile
+      &-sm {
+        @include image-facet-mobile-spacing();
+      }
+    }
+
+    &--form {
+      @include fd-flex(column);
+    }
+
+    &--key-value {
+      .#{$block}__object-status {
+        font-family: var(--sapFontFamily);
+        font-size: $fd-object-status-facet-font-size;
```

Comment:
Object Status Large design does have 1.5rem font-size but the font-family and how the status is displayed with the label(horizontal or vertical) is also different:
![image](https://user-images.githubusercontent.com/53509521/106841892-ff70b180-66c8-11eb-9b4a-f9971a88578d.png)

![image](https://user-images.githubusercontent.com/53509521/106841810-cd5f4f80-66c8-11eb-8a80-3df73b6cd0a8.png)

the reason why extra styles were added to object number is because when used as is, the alignment was looking off:
![image](https://user-images.githubusercontent.com/53509521/106841466-2ed2ee80-66c8-11eb-8d6d-50298e965e43.png)